rosbag2_storage_sqlite3/include/rosbag2_storage_sqlite3/sqlite_storage.hpp

```diff
@@ -125,13 +125,16 @@ class ROSBAG2_STORAGE_DEFAULT_PLUGINS_PUBLIC SqliteStorage
   RCPPUTILS_TSA_REQUIRES(database_write_mutex_);
   int get_last_rowid();
   int read_db_schema_version();
+  uint64_t get_page_size();
```

Comment:
This is logically:

```suggestion
  uint64_t get_page_size() const;
```

Though I'm not sure if that will work given the accesses to the database.  Something to consider, though.

Reply:
It seems ok to make it const. I will add it.